.gitignore

```diff
@@ -7,7 +7,8 @@
 .vscode
 npm-debug.log
 node_modules
-media/uploads/*
-!media/uploads/3d5c935c.xml
-d3/query-graphs.bundle.js
```

Comment:
removed bundle.js from ignore, but then changed query-graphs.html to reference min.js?

i think we stopped pushing all (?) build artifacts at some point, but then we needed min.js in the repo for github-pages, which uses query-graphs.tlv.html

so i think we need min.js, but not bundle.js?

Reply:
yes, you are right:
* we should keep checking in min.js for the github-pages
* bundles.js should remain in the .gitinore

Adjusted my commit